### PR TITLE
renames slides components

### LINKED ISSUE #1: Rename all component to use slides-prefix and reconfigure angular-cli
something which is not `app` but not as long as presentation?

### PR DESCRIPTION
**Pull request by tessaSAC**
Fixes #1

### CODE REVIEW

File: .angular-cli.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
       "test": "test.ts",
       "tsconfig": "tsconfig.app.json",
       "testTsconfig": "tsconfig.spec.json",
-      "prefix": "app",
+      "prefix": "slides",
```
Comment (tessaSAC) [inline .angular-cli.json]: Hopefully this was the way to go 😋💦

File: tslint.json [hunk 2]
```diff
@@ -98,8 +98,8 @@
       "check-type"
     ],
 
-    "directive-selector": [true, "attribute", "app", "camelCase"],
-    "component-selector": [true, "element", "app", "kebab-case"],
+    "directive-selector": [true, "attribute", "slides", "camelCase"],
+    "component-selector": [true, "element", "slides", "kebab-case"],
```
Comment (tessaSAC) [inline tslint.json]: and these two as well 👀💦💦

**Comment by tessaSAC** [comment]
PTAL, thanks! 🐶

**Comment by kirjs** [comment]
My build passes locally, pushing this in

**Pull request merged**